src/mmap.cpp

```diff
@@ -224,24 +224,25 @@ namespace {
 			;
 	}
 
+	auto create_file(const wstring_view name, open_mode_t const mode)
+	{
+		return CreateFileW(name.data()
```

Comment:
same thing applies here. I think it would be better to take the `std::wstring` here instead, to have it moved in. Then you can call `.c_str()` on that